x/employeestore/module.go

```diff
@@ -0,0 +1,118 @@
+package nameservice
+
+import (
+	"encoding/json"
+
+	"github.com/gorilla/mux"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/bank"
+	"github.com/cosmos/sdk-tutorials/nameservice/x/nameservice/client/cli"
```

Comment:
Namespace client was improted and all the code in module is referring to namespace rather than employee store.